quay-enterprise/configure-machines.md

```diff
@@ -1,15 +1,58 @@
-# Configure Machines for Enterprise Registry
+# Configure Machines for Quay Enterprise
 
-The Enterprise Registry allows you to create teams and user accounts that match your existing business unit organization. A special type of user, a robot account, is designed to be used programatically by deployment systems and other pieces of software. Robot accounts are commonly configured with read-only access to an organizations repositories.
+Quay Enterprise allows you to create teams and user accounts that match your existing business unit organization. A special type of user, a robot account, is designed to be used programatically by deployment systems and other pieces of software. Robot accounts are commonly configured with read-only access to an organizations repositories.
 
 This guide we will assume you have the DNS record `registry.example.com` configured to point to your Enterprise Registry.
 
 ## Credentials
 
-Each CoreOS machine needs to be configured with the username and password for a robot account in order to deploy your containers. Docker looks for configured credentials in a `.dockercfg` file located within the user's home directory. You can download this file directly from the Enterprise Registry interface. Let's assume you've created a robot account called `myapp+deployment`.
+Each CoreOS machine needs to be configured with the username and password for a robot account in order to deploy your containers. Docker looks for configured credentials in a `.dockercfg` file located within the user's home directory. You can download this file directly from the Quay Enterprise interface. Let's assume you've created a robot account called `myapp+deployment`.
 
 Writing the `.dockercfg` can be specified in [cloud-config](https://coreos.com/os/docs/latest/cloud-config.html) with the write_files parameter, or created manually on each machine.
 
+### Kubernetes Pull Secret
+
+If you are using Quay Enterprise in conjunction with a Kubernetes or Tectonic cluster, it's easiest to use the built in secret distribution method. This method allows for you to use different sets of robot accounts on a per-app basis, and also allows for them to be updated or rotated at any time across all machines in the cluster.
+
+An "Image Pull Secret" is a special secret that Kubernetes will use when pulling down the containers in a pod. It is a base64-encoded Docker config file. Here's an example:
+
+```sh
+$ cat ~/.dockercfg | base64
```

Comment:
ugh, I suppose not -
https://developer.apple.com/library/prerelease/mac/documentation/Darwin/Reference/ManPages/man1/base64.1.html

base64 < ~/.dockercfg should though.

On Thu, Nov 12, 2015 at 5:40 PM, Rob Szumski <notifications@github.com>
wrote:

> In quay-enterprise/configure-machines.md
> <https://github.com/coreos/docs/pull/669#discussion_r44741434>:
>
> >
> >  Writing the `.dockercfg` can be specified in [cloud-config](https://coreos.com/os/docs/latest/cloud-config.html) with the write_files parameter, or created manually on each machine.
> >
> > +### Kubernetes Pull Secret
> > +
> > +If you are using Quay Enterprise in conjunction with a Kubernetes or Tectonic cluster, it's easiest to use the built in secret distribution method. This method allows for you to use different sets of robot accounts on a per-app basis, and also allows for them to be updated or rotated at any time across all machines in the cluster.
> > +
> > +An "Image Pull Secret" is a special secret that Kubernetes will use when pulling down the containers in a pod. It is a base64-encoded Docker config file. Here's an example:
> > +
> > +```sh
> > +$ cat ~/.dockercfg | base64
>
> does this work on both mac/linux?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/docs/pull/669/files#r44741434>.
>